lib/test_cleanup.rbi

```diff
@@ -1,7 +0,0 @@
-# typed: strict
-
-module Homebrew
-  class TestCleanup
-    include Kernel
```

Comment:
It shouldn't be necessary `include Kernel` in a class (`Class` is a subclass of `Object`, which includes `Kernel`… 🤕)